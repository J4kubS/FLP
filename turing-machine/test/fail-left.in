S a F L
a
