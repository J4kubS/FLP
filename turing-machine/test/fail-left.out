Simulation has abnormally terminated. Possible reasons are:
 * Attempted to move the head to the left from the first cell
 * Final state could not be reached
